packages/components/src/components/carousel/carousel-card.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Card = ({ children, width }) => (
```

Comment:
Can we use the [Card](packages/components/src/components/card/card.jsx) component for this?

Reply:
@reza-fs we can but I think, by the current implementation, it can be used easier. 
the most important point of the Card component is its separation of Header, Content, and footer inside the component.
Currently, I am receiving a general list of components to show in the carousel from the user (in `list` prop), regardless of their content. 
but If I use the Card component, I have to get the Header, Content, and footer of each slide separately that makes the carousel's external API so complex. and If I only insert everything into the `Content` section, using the card component loses its point.